xpra/server/window/window_video_source.py

```diff
@@ -1237,9 +1262,12 @@ def checknovideo(*info):
                      force_reload, ve.get_src_format(), enc_in_format)
             clean = True
         elif ve.get_width()!=enc_width or ve.get_height()!=enc_height:
-            scorelog("check_pipeline_score(%s) change of video input dimensions from %ix%i to %ix%i",
-                     force_reload, ve.get_width(), ve.get_height(), enc_width, enc_height)
-            clean = True
+            if hasattr(ve, 'can_ignore_dim_changes') and ve.can_ignore_dim_changes():
+                pass
+            else:
+                scorelog("check_pipeline_score(%s) change of video input dimensions from %ix%i to %ix%i",
+                        force_reload, ve.get_width(), ve.get_height(), enc_width, enc_height)
+                clean = True
```

Comment:
The `can_ignore_dim_changes` should be added to all the encoders interface after being renamed to a more generic name, ie: `can_compress_size(w, h)`.